collectors/mimecast/utils.js

```diff
@@ -222,13 +226,16 @@ function generateHeaders(authDetails, uri) {
     hmac.end();       // can't read from the stream until you call end()
     signature = hmac.read().toString('base64'); 
 
-    return {
+    let returnObj = {
         "Authorization": `MC ${authDetails.accessKey}:${signature}`,
         "x-mc-app-id": authDetails.appId,
         "x-mc-date": hdrDate,
         "x-mc-req-id": requestId,
         "Content-Type": 'application/json'
     };
+
+    AlLogger.debug('MIME000012 url header: ', returnObj);
```

Comment:
please exclude `Authorization ` header from debug log

Reply:
Done, @kkuzmin please review it again.